### PR TITLE
Added JadoPado, One of the leading E-Commerce platforms in Middle East in showcase page

### PR DESCRIPTION
**Pull request by mohammedsabi**
One of the leading E-Commerce/Marketplace in Middle East & Worldwide "JadoPado" developed Android & iOS apps using react-native & website using react. In this commit added JadoPado to the showcase page.

Here are the links to JadoPado apps

iOS: https://itunes.apple.com/ao/app/jadopado/id1149279535
Android: https://play.google.com/store/apps/details?id=com.jadopado.buyer
Website: https://jadopado.com

Articles:
https://www.entrepreneur.com/article/287838
https://www.entrepreneur.com/article/247985

### CODE REVIEW

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -80,6 +80,15 @@ var featured = [
     infoTitle: 'Flipkart Buys Fashion E-tailer Myntra To Fight Amazon',
   },
   {
+    name: 'JadoPado',
+    icon: 'https://lh3.googleusercontent.com/VX8CXya2KmFc-n9aPChiUpzIg2IkuzudWy76OmFKNMu2eQDFq9xFdDMR-U1E7Sa88eUs=w300-rw',
+    linkAppStore: 'https://itunes.apple.com/ao/app/jadopado/id1149279535',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.jadopado.buyer',
+    infoLink: ' https://www.entrepreneur.com/article/287838',
+    infoTitle: `Infographic: JadoPado's Middle East E-Commerce Seller Insights`,
+    defaultLink: 'https://itunes.apple.com/ao/app/jadopado/id1149279535',
```
Comment (hramos) [inline website/src/react-native/showcase.js]: Remove `defaultLink`, it is not needed here.

File: website/src/react-native/showcase.js [hunk 2]
```diff
@@ -80,6 +80,15 @@ var featured = [
     infoTitle: 'Flipkart Buys Fashion E-tailer Myntra To Fight Amazon',
   },
   {
+    name: 'JadoPado',
+    icon: 'https://lh3.googleusercontent.com/VX8CXya2KmFc-n9aPChiUpzIg2IkuzudWy76OmFKNMu2eQDFq9xFdDMR-U1E7Sa88eUs=w300-rw',
+    linkAppStore: 'https://itunes.apple.com/ao/app/jadopado/id1149279535',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.jadopado.buyer',
+    infoLink: ' https://www.entrepreneur.com/article/287838',
```
Comment (hramos) [inline website/src/react-native/showcase.js]: Do you have a technical blog post about React Native that we could use here instead?

**Comment by mohammedsabi** [comment]
Sure. I'l make the changes. Thanks for contacting.

**Comment by mohammedsabi** [comment]
@hramos

**Comment by hramos** [comment]
Thanks for sending this pull request. According to the guidelines in that file, "infoLink" must be one of:

 1/ An English-language news article discussing the app, built either by a funded startup or for a public company
2/ An English-language technical post on a funded startup or public company blog discussing React Native

It does not seem like the article linked here covers React Native.

**Comment by hramos** [comment]
This is currently failing to pass tests as it is breaking the JavaScript array.

**Pull request closed without merging**